security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -264,9 +265,13 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 				ctx = stream.Context()
 				t, err := getCredentialToken(ctx)
 				if err != nil {
-					sdsServiceLog.Errorf("%s Close connection. Failed to get credential token from "+
-						"incoming request: %v", conIDresourceNamePrefix, err)
-					return err
+					tok, err := ioutil.ReadFile("./var/run/secrets/tokens/istio-token")
```

Comment:
We should only do this in side-car mode. Falling back to the local JWT in DaemonSet mode will cause the node agent to send its own JWT to the CA.

Reply:
Good point ! Will fix it ASAP.